src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,421 @@
+package duke;
+
+import duke.Exceptions.RemoveException;
+import duke.Exceptions.TaskException;
+import duke.Exceptions.TimeException;
+import duke.Exceptions.DoneUndoException;
+import duke.tasks.Deadlines;
+import duke.tasks.Event;
+import duke.tasks.ToDo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+
+public class Duke {
+    public static final int TIME_COMMAND = 4;
+    public static final int DEADLINE_LENGTH = 13;
+    public static final int EVENT_LENGTH = 10;
+    public static final int TODO_LENGTH = 9;
+
+    public static ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
Something to ponder about: Is class-level variable and access modifier for tasks appropriate in this case? This tasks could be augmented by any other classes and your duke bot might not work well as intended.